File: python/difference-of-squares/difference_of_squares.py
import numpy as np
def square_of_sum(count):
    return np.power(sum(range(1,count+1)),2)


def sum_of_squares(count):
    return np.power(range(1,count+1),2).sum()


def difference(count):
    return square_of_sum(count)-sum_of_squares(count)

File: cpp/difference-of-squares/difference_of_squares.cpp
#include "difference_of_squares.h"

namespace squares{
    int square_of_sum(int num){
        int total = 0;
        for (int i = 1; i < num+1; ++i){
            total += i;
        }

        return int(pow(total,2));
    }

    int sum_of_squares(int num){
            int total = 0;
            for (int i = 1; i < num+1; ++i){
                total += int(pow(i,2));
            }
            return total;
    }

    int difference(int num){
        return square_of_sum(num)-sum_of_squares(num);
    }
}
